shared/api/models/DateModel.ts

```diff
@@ -127,6 +128,17 @@ class DateModel {
     return dateTime.plus({ minutes: this._offset - dateTime.offset })
   }
 
+  private currentDateToRrule(): Date {
+    // rrule is not correctly handling timezones and is expecting dates in "local time", i.e. in the timezone of dstart
+    // Therefore we have to manually apply timezone offset, e.g. for daylight savings time
+    // https://github.com/jkbrzt/rrule#important-use-utc-dates
+    // Also include dates happening at the moment by subtracting the duration
+    return DateTime.now()
+      .minus(this._duration)
```

Comment:
Why are we subtracting the duration here?

Reply:
to show events that already started but didn't end yet

Reply:
Ah, nice, thanks!